app/screens/channel/channel_post_list/channel_post_list.js

```diff
@@ -20,7 +20,7 @@ import PostList from 'app/components/post_list';
 import PostListRetry from 'app/components/post_list_retry';
 
 const {View: AnimatedView} = Animated;
-const {height: deviceHeight, width: deviceWidth} = Dimensions.get('window');
+const {width: deviceWidth} = Dimensions.get('window');
```

Comment:
can we move this cause once we add the `landscape` support in #909 this might cause problems if we don't recalculate

Reply:
What's the preferred way of doing this now?

Reply:
you could either do what is being done in the #909 PR where we listen for orientation changes and update the value in the state or you could grab the value whenever the components renders, I don't think we have a preferred way but definitely we can have it as a constant that gets set one the file loads as it will never change the value

Reply:
I'll go for the changing it on render since that's probably the cleanest. Something else we could do in the future is include it in the view store, but that could be done later